prow/kube/prowjob_test.go

```diff
@@ -18,25 +18,27 @@ package kube
 
```

Comment:
(also this file)

Reply:
I wanted this change to be a NOOP so we can do that in a follow-up